src/test/java/com/qdesrame/openapi/test/BackwardCompatibilityTest.java

```diff
@@ -54,31 +58,33 @@ public void testApiReadWriteOnlyPropertiesChanged() {
 
   @Test
   public void testApiResponseDefaultAdded() {
-      OpenAPI spec = OpenApiCompare.getOpenApiParser().read(OPENAPI_DOC5);
-      OpenAPI specWithDefault = specWithDefaultResponse(OPENAPI_DOC5, new ApiResponse().description("hi"));
-      assertOpenApiBackwardIncompatible(spec, specWithDefault);
+    OpenAPI spec = OpenApiCompare.getOpenApiParser().read(OPENAPI_DOC5);
+    OpenAPI specWithDefault = specWithDefaultResponse(OPENAPI_DOC5, new ApiResponse().description("hi"));
+    assertOpenApiBackwardCompatible(spec, specWithDefault, true);
   }
 
   @Test
   public void testApiResponseDefaultRemoved() {
-      OpenAPI spec = OpenApiCompare.getOpenApiParser().read(OPENAPI_DOC5);
-      OpenAPI specWithDefault = specWithDefaultResponse(OPENAPI_DOC5, new ApiResponse().description("hi"));
-      assertOpenApiBackwardIncompatible(specWithDefault, spec);
+    OpenAPI spec = OpenApiCompare.getOpenApiParser().read(OPENAPI_DOC5);
+    OpenAPI specWithDefault = specWithDefaultResponse(OPENAPI_DOC5, new ApiResponse().description("hi"));
+    assertOpenApiBackwardIncompatible(specWithDefault, spec);
   }
 
   @Test
   public void testApiResponseDefaultChanged() {
-    OpenAPI specWithDefault1 = specWithDefaultResponse(OPENAPI_DOC5, new ApiResponse().description("hi"));
-    OpenAPI specWithDefault2 = specWithDefaultResponse(OPENAPI_DOC5, new ApiResponse().description("hey"));
```

Comment:
so just changing the description is "compatible" now, right?

Reply:
Yep!